torBot.py

```diff
@@ -130,6 +134,11 @@ def main(conn=False):
     if args.update:
         updater.updateTor()
         exit()
+    if args.database:
+        check = savedb.check_db_options()
```

Comment:
Why not handle this within `savedb`?

Reply:
did this to exit from torbot.py instead from savedb.py, as discussed internally

Reply:
Yes but what I meant was that the function being called from `savedb.py` should return an error value and that error value should be handled in `torBot.py`. You should only be calling one function from `savedb.py`. Does that make a little more sense?

Reply:
Yea I got the point now.But I'm afraid, then we will be checking for the database parameters at the end, when torbot has finished collecting the results. Since the other function requires the *links* from torbot.py, we will have to do the call, check for parameters at that point of that time. Let me know if that will be ok, to go ahead with it.

Reply:
OK that makes sense, we'll probably move all of the error checking code to it's own function later. The only other picky thing I have is that instead of assigning the return value for `check_db_options()`, why not just use it directly? ```if savedb.check_db_options() == 1```